packages/playwright-test/types/testReporter.d.ts

```diff
@@ -584,6 +584,11 @@ export interface TestError {
    */
   message?: string;
 
+  /**
+   * Source code snippet with highlighted error.
+   */
+  snippet?: string;
```

Comment:
Let's use this snippet for raw/html right away, to make sure we like it.

Reply:
As discussed offline, the only part where we read files to create snippets in raw reporter is for steps snippets and the plans is to ditch step snippets altogether in the merged reports.